[2024-04-11T15:52:02.774+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-11T15:52:02.783+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_gg_bigquery_v07.create_dataset scheduled__2024-04-09T17:00:00+00:00 [queued]>
[2024-04-11T15:52:02.787+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_gg_bigquery_v07.create_dataset scheduled__2024-04-09T17:00:00+00:00 [queued]>
[2024-04-11T15:52:02.787+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-11T15:52:02.792+0000] {taskinstance.py:2327} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2024-04-09 17:00:00+00:00
[2024-04-11T15:52:02.797+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7696) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-11T15:52:02.798+0000] {standard_task_runner.py:63} INFO - Started process 7708 to run task
[2024-04-11T15:52:02.797+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_gg_bigquery_v07', 'create_dataset', 'scheduled__2024-04-09T17:00:00+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_gg_bigquery.py', '--cfg-path', '/tmp/tmpwsd9i7lf']
[2024-04-11T15:52:02.798+0000] {standard_task_runner.py:91} INFO - Job 71: Subtask create_dataset
[2024-04-11T15:52:02.825+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_gg_bigquery_v07.create_dataset scheduled__2024-04-09T17:00:00+00:00 [running]> on host a4f2bd86ee13
[2024-04-11T15:52:02.862+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='duy' AIRFLOW_CTX_DAG_ID='dag_with_gg_bigquery_v07' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-09T17:00:00+00:00'
[2024-04-11T15:52:02.863+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-11T15:52:02.875+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-11T15:52:02.875+0000] {base.py:84} INFO - Using connection ID 'gcp' for task execution.
[2024-04-11T15:52:02.914+0000] {bigquery.py:470} INFO - datasetId was not specified in `dataset_reference`. Will use default value retail.
[2024-04-11T15:52:02.914+0000] {bigquery.py:470} INFO - projectId was not specified in `dataset_reference`. Will use default value ***-projecct.
[2024-04-11T15:52:02.914+0000] {bigquery.py:481} INFO - Creating dataset: retail in project: ***-projecct 
[2024-04-11T15:52:03.835+0000] {bigquery.py:2045} INFO - Dataset retail already exists.
[2024-04-11T15:52:03.847+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-11T15:52:03.854+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_gg_bigquery_v07, task_id=create_dataset, execution_date=20240409T170000, start_date=20240411T155202, end_date=20240411T155203
[2024-04-11T15:52:03.868+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-11T15:52:03.875+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296 AirflowProviderDeprecationWarning: Call to deprecated class BigQueryExecuteQueryOperator. (This operator is deprecated. Please use `BigQueryInsertJobOperator`.)
[2024-04-11T15:52:03.883+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-11T15:52:03.884+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
